Backend/NoCode.FlowerShop.Contracts/Employees/Authentication/AdministratorAuthenticationResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace NoCode.FlowerShop.Contracts.Employees.Authentication;
+
+public record AdministratorAuthenticationResponse(
+    Guid Id,
```

Comment:
Senesnėje versijoje prie CustomerAuthenticationResponse gražinamas ne GUID, bet int.

Reply:
Nice catch! Will fix that.